README.md

```diff
@@ -255,5 +256,6 @@ release in which the functionality was marked *Deprecated*.
 [exporter-stackdriver]: https://godoc.org/contrib.go.opencensus.io/exporter/stackdriver
 [exporter-zipkin]: https://godoc.org/go.opencensus.io/exporter/zipkin
 [exporter-jaeger]: https://godoc.org/go.opencensus.io/exporter/jaeger
-[exporter-xray]: https://github.com/census-instrumentation/opencensus-go-exporter-aws
+[exporter-xray]: https://github.com/census-ecosystem/opencensus-go-exporter-aws
```

Comment:
Did you mean to prefix `https://godoc.org/` for these?

Reply:
No, I just changed the GH org.